Example/ViewController.swift

```diff
@@ -139,6 +165,15 @@ class ViewController: UIViewController {
         presentActionsAlertController()
     }
     
+    func observeSpeedNotifications() {
+        NotificationCenter.default.addObserver(self, selector: #selector(updateSpeedLimitView(notification:)), name:.passiveLocationDataSourceDidUpdate, object: nil)
```

Comment:
The ViewController extension in ViewController+FreeDrive.swift already observes `passiveLocationDataSourceDidUpdate` (and removes the observation at the right time), so we should consolidate this functionality within `subscribeForFreeDriveNotifications()` and `didUpdatePassiveLocation(_:)`.